### PR TITLE
Added loading indicator

### PR DESCRIPTION
**Pull request by MarkGaze**
Added a loading indicator whilst the dashboard is getting the data from VSTS

### CODE REVIEW

File: src/components/main/main.tsx [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
+    });
   }
 
   public render() {
-    const builds = this.state
+    const builds = this.state && this.state.Builds
       ? this.state.Builds.map(b => <Build key={b.Name + b.BuildNumber} buildName={b.Name} buildNumber={b.BuildNumber} status={b.Status} time={b.EndTime} />)
       : [];
     const halfLength = Math.ceil(builds.length / 2);
+    if(this.state.Loading) {
```
Comment (benmatselby) [inline src/components/main/main.tsx]: Is the linter OK with no space between if and ( ?

File: src/components/main/main.tsx [hunk 2]
```diff
@@ -1,20 +1,31 @@
...
+      return (
+      <div className="loader">
+        <Spinner name="cube-grid" color="white"/>
+      </div>);
+    }
+    else {
```
Comment (benmatselby) [inline src/components/main/main.tsx]: Is the linter OK with `} else {` not being on the same line?

File: src/components/main/main.tsx (hunk 1 above)
Comment (MarkGaze) [reply src/components/main/main.tsx]: I seem to be having issues with my linter, will fix and run the checks. Might be worth adding this as a check to CircleCI too.

File: src/components/main/main.tsx (hunk 2 above)
Comment (MarkGaze) [reply src/components/main/main.tsx]: As above

File: src/components/main/main.tsx (hunk 1 above)
Comment (MarkGaze) [reply src/components/main/main.tsx]: In fact, we don't seem to be running the build on CI... only tests

**Comment by MarkGaze** [comment]
The linter didn't seem to moan but I've fixed the formatting with Prettier anyway

**Comment by MarkGaze** [comment]
Rebased the commits @benmatselby

**Pull request merged**